lib/util/object-to-string.js

```diff
@@ -8,11 +8,11 @@ import { flatten } from 'ramda';
  * @param {object} filter
  * @returns {string} The filter in a string representation
  * @example
- * convertObjectToString({SIV_ATTRIBUTE: {id: included: [123]}})
+ * convertObjectToString({SIV_ATTRIBUTE: {id: {included: [123]}}})
  * returns: `SIV_ATTRIBUTE(id)==[123]`
  */
 export default function convertObjectToString(filter) {
-  if (filter === null) {
+  if (isNil(filter)) {
```

Comment:
Question: To confirm, this covers both `undefined` and `null`; is that what we want, or should it only be `null`?

Reply:
It's probably fine. When there is already ramda, I think using it is a good idea. If this is not what we need, we can adjust it later.

Reply:
Problem is: the is _technically_ a breaking change, given it used to return `''` for `undefined`, rather than `null`. We don't want to force a major version bump (being good library development citizens), so we should strongly consider keeping the behaviour the same as before.

Reply:
Technically, it used to return `TypeError: Cannot read property 'constructor' of undefined` for `undefined`. I talked to @Yathi and will adjust the implementation though.